src/Surfnet/StepupMiddlewareClientBundle/Configuration/Command/RemoveRaLocationCommand.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+/**
+ * Copyright 2014 SURFnet bv
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+namespace Surfnet\StepupMiddlewareClientBundle\Configuration\Command;
+
+use Surfnet\StepupMiddlewareClientBundle\Command\AbstractCommand;
+
+class RemoveRaLocationCommand extends AbstractCommand
+{
+    /**
+     * @var string
+     */
+    public $authorityId;
```

Comment:
This isn't present on the command in the Middleware. Maybe you copied this from `RevokeRegistrantsSecondFactorCommand`, but it isn't necessary (afaik) to record the authority's ID in the domain.